src/commcare_cloud/ansible/roles/supervisor/tasks/main.yml

```diff
@@ -59,8 +59,7 @@
 
 - name: stop supervisor to pick up configuration changes
   become: yes
-  command: stop supervisord
-  failed_when: no
+  service: name=supervisord state=stopped
```

Comment:
I'm confused why this wouldn't also need a
```
when: which_supervisord.stdout
```
Maybe `state=stopped` doesn't actually require the thing to exist at all?

Reply:
When I ssh onto a machine before supervisor is installed, `stop supervisord` throws an error. But yeah, maybe having it as a service doesn't require it to exist. Should I put the `when: which_supervisord.stdout` line here for completeness?

Reply:
Eh, if it didn't cause problems, I'm fine with it!